### PR TITLE
Fix Test Crahes/Warnings #2

### PR DESCRIPTION
**Pull request by apoorv-2204**

### CODE REVIEW

**Comment by Neylix** [comment]
Awesome !
That will be so nice do not have so much error log during the test, well done !
I still have 2 errors (appearing almost all the time)
image
image

File: test/archethic/bootstrap_test.exs [hunk 1]
```diff
@@ -598,6 +596,22 @@ defmodule Archethic.BootstrapTest do
            }}
       end)
 
+      # Enum.each(Process.list(), fn pid ->
```
Comment (Neylix) [inline test/archethic/bootstrap_test.exs:599]: You can remove these debug logs

**Review by Neylix — CHANGES_REQUESTED** [review]

**Review by Neylix — APPROVED** [review]

File: lib/archethic/utils.ex [hunk 2]
```diff
@@ -60,27 +60,39 @@ defmodule Archethic.Utils do
...
           list(
-            process ::
-              atom()
-              | {process :: atom(), args :: list()}
-              | {process :: atom(), args :: list(), opts :: list()}
+            (process ::
+               any()
+               | {process :: atom(), args :: list()}
+               | {process :: atom(), args :: list(), opts :: list()})
+            | %{id: atom(), start: {process :: atom(), fx :: atom(), args :: list(any())}}
```
Comment (samuelmanzanera) [inline lib/archethic/utils.ex:67]: What is the difference with Supervisor.child_spec ?

**Review by samuelmanzanera — COMMENTED** [review]
I also saw on the GIthub Actions logs some error or warning regarding mem tables and TransactionSubscriber

**Comment by apoorv-2204** [comment]
> I also saw on the GIthub Actions logs some error or warning regarding mem tables and TransactionSubscriber

For that I have dropped async test, we have to rethink about test designs. To make test async

File: lib/archethic/utils.ex (hunk 2 above)
Comment (apoorv-2204) [reply lib/archethic/utils.ex:67]: Cache lru and lru disk are using child specs. And we need to refactor that part with behavior impl. for now we just accept specs as given .

Comment (samuelmanzanera) [reply lib/archethic/utils.ex:67]: Yes but this is define in the `Supervisor.child_spec()` typespec

**Comment by samuelmanzanera** [comment]
I still got some issues in the tests for graphql_tests

image

Also the graphql tests are really slow (~4s)

**Comment by apoorv-2204** [comment]
> I still got some issues in the tests for graphql_tests
> 
> image
> 
> Also the graphql tests are really slow (~4s)

Drop build and retry.
We can have an another pr for it, as so many other issues are still remaining with tests

**Comment by samuelmanzanera** [comment]
> 

I think it's my conf. No worry

**Comment by apoorv-2204** [comment]
> > 
> 
> I think it's my conf. No worry



> > 
> 
> I think it's my conf. No worry



> > 
> 
> I think it's my conf. No worry

ok BOSS

**Comment by samuelmanzanera** [comment]
> > I think it's my conf. No worry
> 
> ok BOSS

Can you fix the conflicts before merge ?

**Comment by apoorv-2204** [comment]
> > > I think it's my conf. No worry
> > 
> > 
> > ok BOSS
> 
> Can you fix the conflicts before merge ?

Done

**Pull request merged**